raiden/tests/conftest.py

```diff
@@ -5,36 +5,37 @@
 
 import signal  # isort:skip # noqa
 import pytest  # isort:skip
-
-# Execute these before the other imports because rewrites can't work after the
-# module has been imported.
-pytest.register_assert_rewrite("raiden.tests.utils.eth_node")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.factories")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.messages")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.network")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.protocol")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.smartcontracts")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.smoketest")  # isort:skip
-pytest.register_assert_rewrite("raiden.tests.utils.transfer")  # isort:skip
-
 import datetime
 import os
 import re
+import subprocess
 import sys
 import tempfile
 from pathlib import Path
 
 import gevent
 from _pytest.pathlib import LOCK_TIMEOUT, ensure_reset_dir, make_numbered_dir_with_cleanup
 from _pytest.tmpdir import get_user
+from pkg_resources import parse_version
 
-from raiden.constants import EthClient
+from raiden.constants import HIGHEST_SUPPORTED_GETH_VERSION, EthClient
 from raiden.log_config import configure_logging
 from raiden.tests.fixtures.blockchain import *  # noqa: F401,F403
 from raiden.tests.fixtures.variables import *  # noqa: F401,F403
 from raiden.tests.utils.transport import make_requests_insecure
 from raiden.utils.cli import LogLevelConfigType
 
+# Execute these before the other imports because rewrites can't work after the
+# module has been imported.
+pytest.register_assert_rewrite("raiden.tests.utils.eth_node")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.factories")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.messages")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.network")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.protocol")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.smartcontracts")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.smoketest")  # isort:skip
+pytest.register_assert_rewrite("raiden.tests.utils.transfer")  # isort:skip
```

Comment:
Should the imports either be done before the other imports, or should the comment be removed?

Reply:
We now have automatic linting. This is what the linter did. I would leave it as is.

Reply:
@err508 is correct, please read the comment that is just above the calls or the [relevant documentation](https://doc.pytest.org/en/latest/writing_plugins.html)

Reply:
AAaah @err508 and @hackaugusto I did not check the diff that was generated. Sorry this new black+isort thing confuses me and I don't expect changes I did not make.